### PR TITLE
Table Style Parity

### PR DESCRIPTION
**Pull request by brentonmallen1**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Attempts to ensure table styling parity between dark and light modes. The changes were necessary because the appearance of table borders changed when switching between dark and light mode.

### Issues Resolved

https://github.com/luizdepra/hugo-coder/issues/625

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

**Comment by brentonmallen1** [comment]
I haven't run `make release` yet xcode want's me to update to update first 🙄

File: assets/scss/_base_dark.scss [hunk 1]
```diff
@@ -49,11 +49,37 @@
...
+
+  table tr td:first-child,
+  table tr th:first-child {
+    border-left: 0;
+  }
+
+  table tr td:last-child,
+  table tr th:last-child {
+    border-right: 0;
+  }
```
Comment (luizdepra) [inline assets/scss/_base_dark.scss:83]:
```suggestion
}
```
Missing `}` here.

**Comment by luizdepra** [comment]
I fixed a missing `}`. Could please run `make release` now?

**Comment by luizdepra** [comment]
LGTM. Thank you. :)

**Pull request merged**